.github/workflows/kind.yml

```diff
@@ -483,9 +471,4 @@ jobs:
       uses: geekyeggo/delete-artifact@v1
       with:
         name: flow-aggregator-cov
-    - name: Delete antrea-ubuntu
-      if: ${{ needs.build-antrea-image.result == 'success' }}
-      uses: geekyeggo/delete-artifact@v1
-      with:
-        name: antrea-ubuntu
```

Comment:
Is it delete because there is no artifact with name "antrea-ubuntu"?

Reply:
yes, all steps will use `antrea-ubuntu-cov` image since they are almost the same, no need to build another one and delete it.